### ISSUE TITLE
1.8.9: crash after start (Linux Ubuntu 20.04)

### ISSUE THREAD

**Issue by dada513** [N3ROO/Bettercolors#89]
**Describe the bug:**

After launch of minecraft with bettercolors, it shows the menu screen for a split second then crashes with:
```
The game crashed whilst unexpected error
Error: java.lang.IndexOutOfBoundsException: 260
Exit Code: 255
```

**To Reproduce (or what were you doing when it happened):**
Installed forge 1.8.9
dropped bettercolors 1.8.9 to mods folder

**Screenshots (if applicable):**
![img](https://i.imgur.com/KLayFZ1.png)

**Crash log:**

https://pastebin.com/raw/8C7pRSUQ

- **Have you tried running the mod using the option debug=true**: yes / **no**
- **Have you tried deleting the config files? (minecraft/config/bc_filename.properties)**: **yes** / no

**Comment by N3ROO** [comment]
Did you use a newer version of Bettercolors in the same minecraft directory? I suspect that the minecraft/config/bc_default.properties file has the toggle key ID of 260 (the one for MC1.16.5 whereas you are using 1.8.9 and it should be 210)

If you want to keep the settings that you are currently using, then:
- Go to your minecraft directory
- Go to config/
- Open bc_default.properties with notepad, and edit the line where it says "toggle_key=260" to "toggle_key=210"

I will update the mod later on to handle this issue automatically. It's due to the fact that key codes are not the same on different minecraft versions

**Comment by dada513** [comment]
Whoops. I already deleted the config foler before reporting the issue, but now that I looked at the mc folder it seems that something really werid happened and the files were named config/ instead being in the config folder. Screenshot attached below

![image](https://user-images.githubusercontent.com/49000790/107769174-ddfe6e00-6d37-11eb-96a5-76b822ceb7f2.png)

What I belive is causing this is the use of `\` instead of `/` in paths. MS uses \ while linux uses /, causing it to not write them to the folder.

After deleting the werid artifacts, the issue is solved. Thanks.
